generative_ai/gemini_tuning.py

```diff
@@ -69,6 +69,7 @@ def gemini_tuning_advanced() -> sft.SupervisedTuningJob:
         # The following parameters are optional
         validation_dataset="gs://cloud-samples-data/ai-platform/generative_ai/sft_validation_data.jsonl",
         epochs=4,
+        adapter_size=4,
```

Comment:
Hi @liulehui - Thanks for the PR. Q. Why is it important to define this param? What is the benefit of this?

Reply:
Hi Sampath, thanks for the comment!
For this tuning, we have a basic tuning example and another advanced example here. this is for customer observability that they can pass this field in the advance request.
e.g. the python example should be the same as the one with curl in the API doc: https://cloud.google.com/vertex-ai/generative-ai/docs/model-reference/tuning#advanced_use_case